src/core/utils/taborder.js

```diff
@@ -0,0 +1,64 @@
+import SearchParams from '../../ui/dom/searchparams';
```

Comment:
Now that I have a bit more context on the change, I agree with @creotutar, this should probably live in the UI portion of the SDK. The Core is reserved for those parts of the SDK that aren't UI-specific. For instance, we have classes in Core for interacting with the various back-ends. We also keep our data models here. These helpers are much more related to the front-end, UI portion of the SDK. They are only used to help two Components render the proper links. 

Maybe we add this file under the src/ui/tools directory?